src/functions-reference/bounded_discrete_distributions.Rmd

```diff
@@ -155,6 +155,132 @@ The log binomial probability mass of n successes in N trials given
 logit-scaled chance of success alpha dropping constant additive terms
 `r since("2.25")`
 
+## Binomial-logit generalized linear model (Logistic Regression) {#binomial-logit-glm}
+
+Stan also supplies a single function for a generalized linear model
+with Binomial likelihood and logit link function, i.e. a function
+for logistic regression with aggregated outcomes. This provides a more efficient
+implementation of logistic regression than a manually written
+regression in terms of a Binomial likelihood and matrix
+multiplication.
+
+### Probability mass function
+
+Suppose $N \in \mathbb{N}$, $x\in \mathbb{R}^{n\cdot m}, \alpha \in \mathbb{R}^n, \beta \in \mathbb{R}^m$, and $n \in
+\{0,\ldots,N\}$.  Then \begin{align*}
+&\text{BinomialLogitGLM}(n~|~N, x, \alpha, \beta) = \text{Binomial}(n~|~N,\text{logit}^{-1}(\alpha_i + x_i\cdot
+\beta)) \\ &= \binom{N}{n} \left( \text{logit}^{-1}(\alpha_i + \sum_{1\leq j\leq m}x_{ij}\cdot \beta_j) \right)^{n}  \left( 1 -
+\text{logit}^{-1}(\alpha_i + \sum_{1\leq j\leq m}x_{ij}\cdot \beta_j) \right)^{N - n}.  \end{align*}
+
+### Sampling statement
+
+`n ~ ` **`binomial_logit_glm`**`(N, x, alpha, beta)`
+
+Increment target log probability density with `binomial_logit_glm_lupmf(n | N, x, alpha, beta)`.
+`r since("2.34")`
+
+<!-- real; binomial_logit_glm ~; -->
+\index{{\tt \bfseries binomial\_logit\_glm }!sampling statement|hyperpage}
+
+### Stan Functions
+
+<!-- real; binomial_logit_glm_lpmf; (int n | int N, matrix x, real alpha, vector beta); -->
+\index{{\tt \bfseries binomial\_logit\_glm\_lpmf  }!{\tt (int n \textbar\ int N, matrix x, real alpha, vector beta): real}|hyperpage}
+
+`real` **`binomial_logit_glm_lpmf`**`(int n | int N, matrix x, real alpha, vector beta)`<br>\newline
+The log Binomial probability mass of n given N trials and chance of success
+`inv_logit(alpha + x * beta)`.
+`r since("2.34")`
+
+<!-- real; binomial_logit_glm_lupmf; (int n | int N, matrix x, real alpha, vector beta); -->
+\index{{\tt \bfseries binomial\_logit\_glm\_lupmf  }!{\tt (int n \textbar\ int N, matrix x, real alpha, vector beta): real}|hyperpage}
+
+`real` **`binomial_logit_glm_lupmf`**`(int n | int N, matrix x, real alpha, vector beta)`<br>\newline
+The log Binomial probability mass of n given N trials and chance of success
+`inv_logit(alpha + x * beta)` dropping constant additive terms.
+`r since("2.34")`
+
+<!-- real; binomial_logit_glm_lpmf; (int n | int N, matrix x, vector alpha, vector beta); -->
+\index{{\tt \bfseries binomial\_logit\_glm\_lpmf  }!{\tt (int n \textbar\ int N, matrix x, vector alpha, vector beta): real}|hyperpage}
+
+`real` **`binomial_logit_glm_lpmf`**`(int n | int N, matrix x, vector alpha, vector beta)`<br>\newline
+The log Binomial probability mass of n given N trials and chance of success
```

Comment:
Don't need to say "mass" here---it's just a probability.  Or it's a "log probability mass function" if you want to spell it all out.

Reply:
"probability mass" is used throughout the other function docs:

- [`binomial_lpmf`](https://github.com/stan-dev/docs/blob/56bd4fd76d6cc300df5846dbdf0162bd7318217e/src/functions-reference/bounded_discrete_distributions.Rmd#L59)
- [`bernoulli_lpmf`](https://github.com/stan-dev/docs/blob/56bd4fd76d6cc300df5846dbdf0162bd7318217e/src/functions-reference/binary_distributions.Rmd#L44)
- [`ordered_logistic_lpmf`](https://github.com/stan-dev/docs/blob/56bd4fd76d6cc300df5846dbdf0162bd7318217e/src/functions-reference/bounded_discrete_distributions.Rmd#L588)

Reply:
I'd argue that "probability mass" is not idiomatic in this context because we just call the resulting quantity "probability" not a "probability mass".  It's a "probability mass function" but it returns a probability not a probability mass.    So this is another case where the binomial, Bernoulli, etc. need to be fixed.  No worries if you can't get to it in this PR.